project-base/app/src/DataFixtures/Demo/StoreDataFixture.php

```diff
@@ -85,7 +87,9 @@ private function getDemoData(): array
                 self::ATTR_CITY => 'Ostrava',
                 self::ATTR_POSTCODE => '70200',
                 self::ATTR_COUNTRY => $this->getReference(CountryDataFixture::COUNTRY_CZECH_REPUBLIC, Country::class),
-                self::ATTR_CONTACT_INFO => null,
+                self::ATTR_EMAIL => 'ostrava@shopsys.cz',
```

Comment:
could not these emails have side effects? What if they are (or at some point in the future will be) real? :smile: I would rather go with no-reply

Reply:
And is this a problem like if this email becomes active? This functionality is only to display that on the storefront. And I would like to have those emails unique for every store. If you still think that this would be problem I can change that